### PR TITLE
Request review for Heading features branch

### PR DESCRIPTION
**Pull request by vanina123**
- created the index.html file
- created the style.css file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,66 +58,59 @@ After you're finished please remove all the comments and instructions!
...
-**[your_project__name]** is a...
+**Hello-Mircoverse** is an inspiration to give me more zeal to code and learn more,all learning new techques in solving issues
 
-## 🛠 Built With <a name="built-with"></a>
+## 🛠 Built With HTML and CSS <a name="built-with"></a>
 
-### Tech Stack <a name="tech-stack"></a>
+### Tech Stack HTML & CSS <a name="tech-stack"></a>
 
 > Describe the tech stack and include only the relevant sections that apply to your project.
```
Comment (mckent05) [inline README.md]: - To make your readme more professional, please remove all placeholder text in all sections, i.e texts like `Describe the tech stack and include only the relevant sections that apply to your project.`Please fix this 👍

File: README.md [hunk 2]
```diff
@@ -151,7 +142,7 @@ Example commands:
 ### Install
 
 Install this project with:
```
Comment (mckent05) [inline README.md:102]: - To help make your repo more professional, I suggest you include steps on how to setup your project on a local desktop i.e Install, clone, and navigate into your project. Please fix this 🙏

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @vanina123, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @vanina123, 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linter error.
:heavy_check_mark: Descriptive readme file.
:heavy_check_mark: Well-implemented suggestions.

### Optional suggestions :arrow_down_small: 

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [ ] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "fix bug".` [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**